fence/resources/aws/boto_manager.py

```
@@ -33,24 +33,30 @@ def delete_data_file(self, bucket, guid):
             s3_objects = self.s3_client.list_objects_v2(
                 Bucket=bucket, Prefix=guid, Delimiter="/"
             )
+
             if not s3_objects.get("Contents"):
                 # file not found in the bucket
                 self.logger.info(
-                    "tried to delete GUID {} but didn't find in bucket {}".format(
+                    "tried to delete prefix {} but didn't find in bucket {}".format(
```

Why the change from GUID to prefix?

---

In that instance, the value is now the filepath (a.k.a. prefix) of the thing being deleted, not its GUID. originally, the value was indeed the GUID -- but the s3 code was failing silently and nothing was ever being deleted. i fixed it =D